packages/sdk/src/types/internal/context.ts

```diff
@@ -345,7 +345,7 @@ export class InternalContext {
 			execution.on('protocol.destroy-actors', this.localDestroyActors);
 			execution.on('protocol.user-joined', this.userJoined);
 			execution.on('protocol.user-left', this.userLeft);
-			execution.on('protocol.update-user', this.updateUser);
+			execution.on('protocol.user-update', this.updateUser);

```

Comment:
>user-update [](start = 26, length = 11)

Even though this rename is essentially a noop (it worked before), I like user-update better than update-user, since user-update matches the payload type of the same name.